boot/bootutil/src/image_validate.c

```diff
@@ -32,6 +33,9 @@
 #include <string.h>
 #include <errno.h>
 
+//to make INT_MAX available
```

Comment:
```suggestion
```

Reply:
You've changed some includes, I'm not being funny but that is not a contribution that warrants a copyright added to a file, that's not code, that's not even copyrightable